WMF Framework/Localization/WMFLocalization.h

```diff
@@ -3,7 +3,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 // Don't rename this function. It'll break the script that generates the strings. https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/LoadingResources/Strings/Strings.html
```

Comment:
I haven't dug into what this means but just wanted to highlight this comment warning against renaming this.

Reply:
I believe this is because `genstrings` is run on the code to generate the strings files. (For a while, I owned `genstrings` when I worked on AppKit!)

Only the name of the function is used - all custom localized string functions use the same signature. 

Changing the name of the argument doesn't change the signature of the function, so this should not break anything.